documentation/source/reference/kinetics/stickingcoefficient.rst

```diff
@@ -0,0 +1,22 @@
+**********************************
+rmgpy.kinetics.StickingCoefficient
+**********************************
+
+.. autoclass:: rmgpy.kinetics.StickingCoefficient
+
+    The Sticking coefficient class uses an arrhenius-like expression to determine
```

Comment:
```suggestion
    The Sticking coefficient class uses an Arrhenius-like expression to determine
```

Reply:
Using the current version used by the website (v5.0.2) everything renders correctly. Thanks for the review.